<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agents</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" rowspan="2" colspan="7">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>686</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="chatLable">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chat: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>686</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ModelLabel">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model: </string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2" colspan="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>686</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="createTitle_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Previous agents</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QComboBox" name="modelsComboBox"/>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLabel" name="createTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load agent</string>
     </property>
    </widget>
   </item>
   <item row="10" column="6">
    <widget class="QPushButton" name="startAgentButton">
     <property name="text">
      <string>Start agent</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="6">
    <widget class="QComboBox" name="prevAgentsBox"/>
   </item>
   <item row="8" column="1" colspan="6">
    <widget class="QPlainTextEdit" name="agentRolePlanText"/>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chatCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="agentRoleLabl">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Agent role:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="6">
    <widget class="QPushButton" name="loadAgentButton">
     <property name="text">
      <string>Load agent</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
